parts/k8s/cloud-init/artifacts/cse_main.sh

```diff
@@ -57,6 +57,8 @@ if [[ "${GPU_NODE}" != "true" ]]; then
   cleanUpGPUDrivers
 fi
 
+apt_fix_keys &
```

Comment:
@yangl900 my assumption is that in the expired GPU key context, this would only get called for GPU nodes since `cleanUpGPUDrivers` would have already taken care of removing the nvidia apt source by the time we get to `apt_fix_keys`.

@jackfrancis were you able to verify that that's true?

Reply:
correct